src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -65,41 +64,108 @@ class Widget_List extends Widget_Abstract {
 	];
 
 	/**
-	 * @todo update in TEC-3612 & TEC-3613
-	 *
 	 * {@inheritDoc}
-	 *
-	 * @var array<string,mixed>
 	 */
-	protected $validate_arguments_map = [
-		'should_manage_url'    => 'tribe_is_truthy',
-		'no_upcoming_events'   => 'tribe_is_truthy',
-		'featured_events_only' => 'tribe_is_truthy',
-		'jsonld_enable'        => 'tribe_is_truthy',
-	];
+	public function get_arguments() {
+		$arguments = $this->arguments;
+
+		$arguments['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
+		// @todo update name once this widget is ready to replace the existing list widget.
+		$arguments['name']                          = esc_html__( 'Events List V2', 'the-events-calendar' );
+		$arguments['widget_options']['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
+
+		// Setup default title.
+		$arguments['title'] = __( 'Upcoming Events', 'the-events-calendar' );
+
+		// Setup admin fields.
+		$arguments['admin_fields'] = $this->get_admin_fields();
+
+		// Add the Widget to the arguments to pass to the admin template.
+		$arguments['widget_obj'] = $this;
+
+		$arguments = wp_parse_args( $arguments, $this->get_default_arguments() );
+
+		return $this->filter_arguments( $arguments );
+	}
 
 	/**
-	 * @todo update in TEC-3612 & TEC-3613
+	 * The function for saving widget updates in the admin section.
 	 *
-	 * {@inheritDoc}
+	 * @param array $new_instance
+	 * @param array $old_instance
+	 *
+	 * @return array The new widget settings.
 	 */
-	public function setup() {
-		$this->setup_view();
+	public function update( $new_instance, $old_instance ) {
+		$instance = $old_instance;
+
+		/* Strip tags (if needed) and update the widget settings. */
+		$instance['title']                = strip_tags( $new_instance['title'] );
```

Comment:
:warning: `strip_tags()` does not strip CSS and JS in between the script and style tags. Use `wp_strip_all_tags()` to strip all tags.

<sub>hash: 78bb6011e5476323acec8ebc3d22dbca</sub>